notes-webapp/src/main/webapp/vue-app/notes/components/NotesOverview.vue

```diff
@@ -555,7 +555,8 @@ export default {
         this.note = data || {};
         this.loadData = true;
         this.currentNoteBreadcrumb = this.note.breadcrumb;
-        notesConstants.PORTAL_BASE_URL = `${notesConstants.PORTAL_BASE_URL.split(this.appName)[0]}${this.appName}/${this.note.id}`;
+        const charsToRemove = notesConstants.PORTAL_BASE_URL.length-notesConstants.PORTAL_BASE_URL.lastIndexOf('/notes');
```

Comment:
Should we use this.appName instead of notes
```suggestion
        const charsToRemove = notesConstants.PORTAL_BASE_URL.length-notesConstants.PORTAL_BASE_URL.lastIndexOf('/'+this.appName);
```

Reply:
resolved in the same commit